src/Nirum/Package/Metadata.hs

```diff
@@ -96,11 +97,12 @@ deriving instance (Ord t, Target t) => Ord (Package t)
 deriving instance (Show t, Target t) => Show (Package t)
 
 packageTarget :: Target t => Package t -> t
-packageTarget Package { metadata = Metadata _ _ _ t } = t
+packageTarget Package { metadata = Metadata _ _ _ _ t } = t
```

Comment:
As `Metadata` get having many fields, we'd better use record syntax:

```haskell
packageTarget Package { metadata = Metadata { target = t } } = t
```

Reply:
I agree with you, I'll fix it soon. Thank you!